assests/javascript/index.js

```diff
@@ -0,0 +1,80 @@
+const baseUrl = "http://127.0.0.1:3030/";
+
+const newTask = document.getElementById("task-input");
+const addTaskIcon = document.getElementById("add-task-icon");
+const taskBox = document.getElementById("task-box");
+const newTaskErrorMassage = document.getElementById("title-error");
+const connectionErrorMassage = document.getElementById("connection-error");
+
+newTask.addEventListener(
+  "keypress",
+  (event) => {
+    if (event.key === "Enter") {
+      setNewTask(newTask.value);
+      newTask.value = "";
+    }
+  },
+  false
+);
+
+addTaskIcon.addEventListener("click", () => {
+  setNewTask(newTask.value);
+  newTask.value = "";
+});
+
+async function setNewTask(task) {
+  removeErrors();
+
+  try {
+    const body = JSON.stringify({ text: task });
+    const headers = { "Content-Type": "application/json" };
+
+    const response = await fetch(`${baseUrl}todos`, {
+      method: "POST",
+      body,
+      headers,
+    });
+
+    const result = await response.json();
```

Comment:
Use destructuring syntax here.

Reply:
so beautifullllll :,(((((((
I hope I did it right